arches_lingo/migrations/0002_load_lingo_lists.py

```diff
@@ -22,6 +23,7 @@ def load_lists(apps, schema_editor):
                 "controlled_lists",
                 "lingo_lists.json",
             ),
+            stdout=io.StringIO(),
```

Comment:
This is just to silence the extra output during migrate ("n objects installed from fixture")